src/utils/backend.js

```diff
@@ -45,6 +45,21 @@ export default class Backend {
   getProfileImage = async (address) => wrapTry(async () => {
     return fetch(`${this.BACKEND_URL}/profile/image/${address}`);
   });
+  setProfileImage = async (address, data, image = true) => wrapTry(async () => {
+    if (image) {
+      return fetch(`${this.BACKEND_URL}/profile/image/` + address, {
```

Comment:
address can go into the string template, also i would make the fetch params to be conditional, and call only one fetch